.travis.yml

```diff
@@ -108,7 +108,7 @@ matrix:
 
     - os: osx
       env: >
-        TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon-hid-sections
+        TOOLCHAIN=android-ndk-r17-api-24-arm64-v8a-clang-libcxx14
```

Comment:
Do we need `android-ndk-r17-api-19-armeabi-v7a-neon-hid-sections` ?

Reply:
OS X could target r17 + api-19 + c++11.

Reply:
Using `android-ndk-r17-api-19-armeabi-v7a-neon-hid-sections` now